src/utils/resources/bootableresources/hooks/useBootableVolumes.ts

```diff
@@ -27,6 +27,7 @@ type UseBootableVolumes = (namespace?: string) => {
 };
 
 const useBootableVolumes: UseBootableVolumes = (namespace) => {
+  console.log('🚀 ~ file: useBootableVolumes.ts:30 ~ namespace:', namespace);
```

Comment:
Do you want to leave these `console.log` ?

Reply:
lol removed